test/CMakeLists.txt

```diff
@@ -412,6 +412,7 @@ add_sdl_test_executable(testcustomcursor SOURCES testcustomcursor.c)
 add_sdl_test_executable(testvulkan NO_C90 SOURCES testvulkan.c)
 add_sdl_test_executable(testoffscreen SOURCES testoffscreen.c)
 add_sdl_test_executable(testpopup SOURCES testpopup.c)
+add_sdl_test_executable(dialog SOURCES testdialog.c)
```

Comment:
Adding more to your pain: Why does the test program name does not contain `test`? I.e. `dialog` instead of usually expected `testdialog` ?

Reply:
I'm not familiar with the test suite in SDL, so I left that for others to modify. I wrote the original (very basic) test called `dialog.c`, hence why I think the name spread across the code, and the file was eventually renamed properly `testdialog.c`, but related names weren't renamed. I'll change this, and I'll check for anything else that might need to be renamed `testdialog`.

Also no worries, the more I learn now, the less I'll have to learn if I open another PR later :)